index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylessheet"
+      href="https://necolas.github.io/normalize.css/8.0.1/normalize.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>Alejandro Orozco's Portfolio</title>
+  </head>
+
+  <body>
+    <header class="nav">
+      <div>
+        <h3 class="header_Logo">My Logo</h3>
+      </div>
+      <div>
+        <nav class="hamburger">
+          <i class="material-icons">menu</i>
+        </nav>
+      </div>
+    </header>
+
+    <section id="main-section">
+      <h2 class="Title">
+        Hey There. <br />
+        I'm Alejandro.
+      </h2>
+      <br />
+      <h3 class="Title-desc">I'm a Software Engineer</h3>
+
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don't hesitate to contact me.
+      </p>
+      <ul class="container2">
+        <li><img src="icons/github.svg" alt="github" /></li>
+        <li><img src="icons/linkedin.svg" alt="LinkedIn" /></li>
+        <li><img src="icons/hand.svg" alt="github" /></li>
+        <li><img src="icons/Twitter.svg" alt="Twitter" /></li>
+        <li><img src="icons/M.svg" alt="M" /></li>
+      </ul>
```

Comment:
To ensure that your headline section matches the template, consider positioning the illustration at the bottom right of the page correctly and implement the correct sizing also.

Your design 
<img width="215" alt="oops" src="https://github.com/OROZCO1623/A.Orozco_Portfolio/assets/84930355/4fe10da5-07fe-41e9-8894-4d9b965f9ced">


figma 
<img width="269" alt="ds" src="https://github.com/OROZCO1623/A.Orozco_Portfolio/assets/84930355/08e793be-3f1a-46c0-b8cf-6d48d2f86e4b">